### ISSUE TITLE
emerge validation issues

### ISSUE THREAD

**Issue by g00g1** [ToeiRei/crowdsec-overlay#5]
Hello, I just added this crowdsec-overlay to my list, did `emaint sync -a` and tried to install package from this overlay, but portage is unhappy:

```
!!! Repository 'crowdsec-overlay' is missing masters attribute in '/var/db/repos/crowdsec-overlay/metadata/layout.conf'
!!! Set 'masters = gentoo' in this file for future compatibility
WARNING: One or more repositories have missing repo_name entries:

	/var/db/repos/crowdsec-overlay/profiles/repo_name

NOTE: Each repo_name entry should be a plain text file containing a
unique name for the repository on the first line.
```

**Comment by g00g1** [comment]
Oh, I found out the reason - it seems like the issue is in `sudo eselect repository add crowdsec-overlay git git://github.com/ToeiRei/crowdsec-overlay`. I have changed `git://` to `https://` and it worked.

**Comment by ToeiRei** [comment]
https may be the better protocol here. - Fixed in [3cf3e4d](https://github.com/ToeiRei/crowdsec-overlay/commit/3cf3e4de019dc2b8909e1f9abaac05516de942de)
